CMakeLists.txt

```diff
@@ -2312,6 +2322,18 @@ endif()
 
 if(HPX_WITH_EXAMPLES)
   add_hpx_pseudo_target(examples)
+  if(HPX_WITH_FETCH_HWLOC AND "${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+    add_custom_target(
+      HwlocDLL ALL
+      COMMAND ${CMAKE_COMMAND} -E make_directory
+              "${CMAKE_BINARY_DIR}/$<CONFIG>/bin/"
+      COMMAND
+        ${CMAKE_COMMAND} -E copy_if_different
+        "${HWLOC_ROOT}/bin/libhwloc-15.dll"
```

Comment:
Shouldn't this depend on `HPX_WITH_HWLOC_VERSION`?

Reply:
I looked into it, but was not able to find any relation between the release version of HWLoc and `libhwloc-15.dll`. As far as I could see [here](https://www-lb.open-mpi.org/software/hwloc/ancient/), it has been in use since HWLoc v2.0.1